docs/reference/commandline/dockerd.md

```diff
@@ -1327,7 +1327,13 @@ This is a full example of the allowed configuration options on Linux:
 	"userns-remap": "",
 	"group": "",
 	"cgroup-parent": "",
-	"default-ulimits": {},
+	"default-ulimits": {
+		"nofile": {
```

Comment:
Maybe we can pick a different one, just because `nofile` is per user and not per-container.... just basically don't want someone to take this as a recommendation of some sort.

Reply:
Isn't that just for `nproc`? I thought `nofile` was per process. If both are per user then we'll want to update some other docs to warn users. Might be good to have a table of each of the ulimit names and a brief description somewhere to avoid confusion.

Reply:
Ah you are right, it is nproc!